third_party/ginkgo/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ ginkgo_load_git_package(
   "-DGINKGO_BUILD_BENCHMARKS=OFF"
   "-DGINKGO_BUILD_REFERENCE=${GINKGO_BUILD_REFERENCE}"
   "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
+  "-DCMAKE_CUDA_ARCHITECTURES=75"
```

Comment:
```suggestion
```
We don't want to hardcode the CUDA architecture. Does the automatic detection still fail?

Reply:
I switched to a newer cmake and switched to "native".